x-pack/plugins/ml/server/models/data_recognizer/modules/security_network/ml/high_count_by_destination_country.json

```diff
@@ -1,14 +1,11 @@
 {
   "description": "Security: Network - Looks for an unusually large spike in network activity to one destination country in the network logs. This could be due to unusually large amounts of reconnaissance or enumeration traffic. Data exfiltration activity may also produce such a surge in traffic to a destination country which does not normally appear in network traffic or business work-flows. Malware instances and persistence mechanisms may communicate with command-and-control (C2) infrastructure in their country of origin, which may be an unusual destination country for the source network.",
-  "groups": [
-    "security",
-    "network"
-  ],
+  "groups": ["security", "network"],
   "analysis_config": {
     "bucket_span": "15m",
     "detectors": [
       {
-        "detector_description": "high_non_zero_count by \"destination.geo.country_name\"",
+        "detector_description": "Detection high count by country.",
```

Comment:
Any reason why you have gone with `Detection` in this description rather than e.g. `Detects high count by country` which would be consistent with the style used elsewhere?

Reply:
Nope, typo on my part. Fixed it!